spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/ops/FloorDiv.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.intel.analytics.bigdl.nn.ops
+
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.utils.Table
+
+import scala.reflect.ClassTag
+
+class FloorDiv[T: ClassTag, D: ClassTag]()(implicit ev: TensorNumeric[T], ev2: TensorNumeric[D])
+  extends Operation[Table, Tensor[D], T]{
+
+  output = Tensor[D]()
+
+  override def updateOutput(input: Table): Tensor[D] = {
+    val input1 = input[Tensor[D]](1)
+    val input2 = input[Tensor[D]](2)
+    output.resizeAs(input1).copy(input1)
+    output.div(input2)
+    output.floor()
+    output
```

Comment:
Not really sure this is correct, because the result of `math.floor(5 / -2)` is `-2` and  the result of `floordiv` should be `-3`

Reply:
It works for float / double but I didn't test Int... Good catch.